core/base/src/TROOT.cxx

```diff
@@ -1750,6 +1750,14 @@ TCollection *TROOT::GetListOfFunctionTemplates()
    return fFuncTemplate;
 }
 
+namespace {
+template <typename FuncRef>
+void AddToListOfGlobals(TListOfDataMembers *lst, const char *name, const char *tname, FuncRef &func)
```

Comment:
This might get you a `FuncRef` of type `*(*)()` at times, because the pointer passed in is a reference to a variable of type function pointer... I'd really use a `T *` as argument! That might also get rid of the Python errors?